### ISSUE TITLE
Update Ace to get latest a11y fixes

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#13592]
### System details

    RStudio Edition : Desktop and Server
    RStudio Version : 2023.09.0-daily+397
    OS Version      : any
    R Version       : any

RStudio 2023.09 was upgraded to Ace 1.19; there have been several a11y improvements made to Ace since then, and we should consider upgrading Ace again soon to pick these up.

## v1.20.0
Add gutter controls to keyboard accessibility mode ([#5146](https://github.com/ajaxorg/ace/issues/5146)) ([5f2face](https://github.com/ajaxorg/ace/commit/5f2face8febe7c260a42bc30c67195f6a5a46085))
Hide Ace content from assistive technologies to improve a11y by [@akoreman](https://github.com/akoreman) in [#5160](https://github.com/ajaxorg/ace/pull/5160)
## v1.23.0
signal gutterkeydown in keyboard accessibility mode ([#5202](https://github.com/ajaxorg/ace/issues/5202)) ([538b18c](https://github.com/ajaxorg/ace/commit/538b18cf86e21c810910d286c820bac399927cd7))
Improve ARIA labelling fold controls ([#5205](https://github.com/ajaxorg/ace/issues/5205)) ([dad5e6f](https://github.com/ajaxorg/ace/commit/dad5e6f6e445b805f7b470ef7892442a83d342e5))
## v1.23.2
bug(autocomplete): fix a11y violations by [@oykuyilmaz](https://github.com/oykuyilmaz) in [#5241](https://github.com/ajaxorg/ace/pull/5241)
## v1.24.0
Allow more lines in hidden textarea to improve screen reader experience on Windows ([#5225](https://github.com/ajaxorg/ace/issues/5225)) ([bccff5a](https://github.com/ajaxorg/ace/commit/bccff5ae21b86004c2a4ab4a86a989c61e728ebf))

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jonvanausdeln** [comment]
Verified on [2023.12.0-daily+237](https://dailies.rstudio.com/version/2023.12.0-daily+237) 
Windows 11, MacOS 12.6

All related desktop automation passed.
